ranch/utils.py

```diff
@@ -385,6 +385,9 @@ def generate_centroid_connectors_shape(zone_loading_node_df):
         columns={"osm_node_id": "u", "model_node_id_x": "A", "model_node_id_y": "B"}
     )
 
+    if ("A" not in new_cc_gdf.columns ) and ("B" not in new_cc_gdf.columns):
+        new_cc_gdf = new_cc_gdf.rename(columns={"X": "A", "Y": "B"})
```

Comment:
X/Y are the coordinates, A/B are the node IDs, what's the meaning of this rename?